test/testAll.cpp

```diff
@@ -67,5 +69,5 @@ int main(int argc, const char * const * argv)
       }
     }
   }
-  return 0;
+  return failures;
```

Comment:
Would be nice to print out a message with the failure count if there are any failures.